conftest.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from enum import Enum
+from supplier import models
+
+
+class TestFields(Enum):
+    USER_NAME_TEST = 'ScrappyKoko'
+    FIRST_NAME_TEST = 'Zohan'
+    LAST_NAME_TEST = 'Dvir'
+    PASSWORD_TEST = 'silkysmooth'
+    USER_TYPE_TEST = 'supplier'
+    BUSINESS_NAME_TEST = 'hair_salon_inc'
+
+
+class Fields(Enum):
+    user_name = 1
+    first_name = 2
+    last_name = 3
+    password = 4
+    user_type = 5
+    business_name = 6
+
+
+@pytest.fixture
+def supplier0():
```

Comment:
Defining this fixture here means that all tests on your repo are going to have access to this fixture without importing anything. I suggest defining it at the file that you need it. If you later find that you need it elsewhere, then move so it's visible where it's needed, rather than everywhere.

Reply:
like the comment above, since supplier is used by the rest of the apps should it be accessible?

Reply:
If you already know it's going to be needed at this level, then it's fine having it here.

Reply:
cool, used for saved_supplier0.